packages/module/patternfly-docs/content/examples/catalogTile.css

```diff
@@ -1,17 +1,13 @@
 .ws-react-e-catalog-tile .catalog-tile-pf:active, .ws-react-e-catalog-tile .catalog-tile-pf:hover, .ws-react-e-catalog-tile .catalog-tile-pf:focus, .ws-react-e-catalog-tile .catalog-tile-pf:visited {
-  color: inherit;
+  color: var(--pf-t--global--text--color--regular);
   text-decoration: none;
 }
-.ws-react-e-catalog-tile .catalog-tile-pf-header {
-  font-size: 16px;
-  font-weight: 400;
-}
-.ws-react-e-catalog-tile .catalog-tile-pf-header .catalog-tile-pf-title {
-  font-weight: 400;
-}
 .ws-react-e-catalog-tile .catalog-tile-pf-header .catalog-tile-pf-subtitle {
```

Comment:
IMO this is a fair enhancement request for PF - a card subtitle. Then we could get rid of all of this, too.